django/api/services/calculate_rebate.py

```diff
@@ -0,0 +1,102 @@
+from ..models.go_electric_rebate_application import GoElectricRebateApplication
+from ..models.household_member import HouseholdMember
+from ..settings import INCOME_REBATES
+from enum import Enum
+
+
+class RebateType(Enum):
+    A = "A"
+    B = "B"
+    C = "C"
+    D = "Not Approved"
+
+
+def check_individual(primary_income):
+    if primary_income is None:
+        return RebateType.D.value
+    primary_income = int(primary_income)
+    if primary_income > INCOME_REBATES.get(RebateType.C.value).get("individual_income"):
+        return RebateType.D.value
+    elif primary_income <= INCOME_REBATES.get(RebateType.A.value).get(
+        "individual_income"
+    ):
+        return RebateType.A.value
+    elif primary_income <= INCOME_REBATES.get(RebateType.B.value).get(
+        "individual_income"
+    ):
+        return RebateType.B.value
+    elif primary_income <= INCOME_REBATES.get(RebateType.C.value).get(
+        "individual_income"
+    ):
+        return RebateType.C.value
+
+
+def check_household(primary_income, secondary_income):
+    if (primary_income is None) | (secondary_income is None):
```

Comment:
I think this needs to be updated for 

"No income data received from CRA for secondary (spouse) applicant = Calculate rebate based on individual income criteria (as above)"